m2m/m2m.py

```diff
@@ -147,75 +148,74 @@ def writeTemplateFiles(self, baseDirectory, foldername):
+        return '%s finished' % foldername
 
     def writeJSONFile(self, baseDirectory, foldername, data):
         baseDirectory = baseDirectory
         foldername = foldername
         writeDirectory = os.path.join(baseDirectory, foldername)
         try:
             os.makedirs(writeDirectory)
-        except:
+        except OSError:
             if os.path.exists(writeDirectory):
                 pass  # no big deal if they exist
             else:
                 raise MetadataConverterException(
-                    "Unable to create the output directory '%s'. " +
-                    "Perhaps you should check permissions?" %
-                    (writeDirectory,))
-        jsonFilename = open(os.path.join(writeDirectory, "metadata.json"), "w")
+                    'Unable to create the output directory %s.'
+                    'Perhaps you should check permissions?' % writeDirectory)
+        jsonFilename = open(os.path.join(writeDirectory, 'metadata.json'), 'w')
         jsonFile = json.dumps(data,
                               sort_keys=True,
                               indent=4,
                               separators=(',', ': ')
                               )
         jsonFilename.write(jsonFile)
         jsonFilename.close()
-        return "%s finished JSON" % foldername
+        return '%s finished JSON' % foldername
 
 
-if "__main__" == __name__:
+if '__main__' == __name__:
 
-    usage = "usage: %prog [options] <csv file>"
+    usage = 'usage: %prog [options] <csv file>'
 
     parser = OptionParser(usage=usage)
 
-    parser.add_option("-m", "--mapping", action="store", type="string",
-                      dest="mapping",
-                      help="Specify the mapping file to use for this CSV")
+    parser.add_option('-m', '--mapping', action='store', type='string',
+                      dest='mapping',
+                      help='Specify the mapping file to use for this CSV')
 
-    parser.add_option("-n", "--row", action="store", type="int",
-                      dest="row",
-                      help="Specify a single row number to process")
+    parser.add_option('-n', '--row', action='store', type='int',
+                      dest='row',
+                      help='Specify a single row number to process')
 
-    parser.add_option("-w", "--write", action="store_true",
-                      dest="write",
-                      help="Write records to files")
+    parser.add_option('-w', '--write', action='store_true',
+                      dest='write',
+                      help='Write records to files')
 
-    parser.add_option("-j", "--json",
-                      action="store_true",
-                      dest="json",
-                      help="UWrite json version of metadata")
+    parser.add_option('-j', '--json',
+                      action='store_true',
+                      dest='json',
+                      help='UWrite json version of metadata')
```

Comment:
It seems like "UWrite" is a typo?

Reply:
Do you mind if I fix it in arg_parse PR that I'm gonna open once this is merged?

Reply:
sounds fine to me